src/Schema/TableDiff.php

```diff
@@ -68,10 +68,30 @@ public function getDroppedColumns(): array
         return $this->droppedColumns;
     }
 
-    /** @return array<string,Column> */
+    /**
+     * @deprecated Use {@see getModifiedColumns()} instead.
+     *
+     * @return array<string,Column>
+     */
     public function getRenamedColumns(): array
     {
-        return $this->renamedColumns;
+        Deprecation::triggerIfCalledFromOutside(
+            'doctrine/dbal',
+            'https://github.com/doctrine/dbal/pull/6080',
+            '%s is deprecated, you should use `getModifiedColumns()` instead.',
+            __METHOD__,
+        );
```

Comment:
You will send an MR against 5.0.x about this, right?

Reply:
The problem I have is this is quite a handy method, it's good to deprecate for external use but it's very useful for tests (as can be seen by the extensive usage), so either we move it into a test class or make it internal

Reply:
extensive usage?

It's used 4 times in tests: https://github.com/search?q=repo%3Adoctrine%2Fdbal%20getRenamedColumns&type=code

I'd say move it to the one test class that uses it.

Reply:
Then it can be moved in this PR, it's meant to be deprecated on 3.9 with the previous PR